app/domain/operations/eligible/build_evidence.rb

```diff
@@ -0,0 +1,89 @@
+# frozen_string_literal: true
+
+require "dry/monads"
+require "dry/monads/do"
+
+module Operations
+  module Eligible
+    # Operation to support eligibility creation
+    class BuildEvidence
+      include Dry::Monads[:do, :result]
+      include ::Operations::Eligible::EligibilityImport[
+                configuration: "evidence_defaults"
+              ]
+
+      # @param [Hash] opts Options to build eligibility
+      # @option opts [<GlobalId>] :subject required
+      # @option opts [<String>]   :evidence_key required
+      # @option opts [<String>]   :evidence_value required
+      # @option opts [Date]       :effective_date required
+      # @option opts [Date]       :evidence_record optional
+      # @option opts [Hash]       :timestamps optional timestamps for data migrations purposes
+      # @return [Dry::Monad] result
```

Comment:
Mentioned elsewhere, but probably would be useful to indicate what the result contents are - here, a hash (vs model). Being able to reference that at a glance from comments up top or in published docs would be helpful.